aptos-move/framework/aptos-stdlib/sources/data_structures/smart_vector.move

```diff
@@ -27,6 +27,13 @@ module aptos_std::smart_vector {
 
     /// Regular Vector API
 
+    /// Create an empty vector using default logic to estimate `inline_capacity` and `bucket_size`, which may be
+    /// inaccurate.
+    /// This is exactly the same as empty() but is more standardized as all other data structures have new().
```

Comment:
If we do not have it, we should likely add an issue for attribute `#[deprecated]`?

Reply:
Good idea! I added #[deprecated] to the other non-standard create function